app/api/metrics.py

```diff
@@ -51,3 +63,61 @@ def getScoreCards(
     teamId = decodeJWT(token)["teamId"]
 
     return microserviceScoreCardService.getByTeamId(teamId)
+"""
+
+# ADD NEW METRIC HERE
+@router.post("/", response_model=schemas.Metric)
+def createMetric(metric: schemas.MetricCreate, metricCrud: crud.CRUDMetric = Depends(dependencies.getMetricsCrud)) -> schemas.Metric:
+    metricObj = metric
+    # change ' ' with '-'
+    metricObj.code = format_code(metric.name)
+    # Stringfying the list of areas to be saved as string 
+    metricObj.area = json.dumps(metric.area)
+    # Handling the datatype of the field type to be integer or boolean 
+    if (metricObj.type != "integer" and metricObj.type != "boolean"):
+        raise ExceptionCustom(status_code=422, detail= "type must be integer or boolean")
+    metricCrud.create(metricObj)
+    raise ExceptionCustom(status_code=200, detail="Success in creating metric")
+
+# Get Any Metric Here By ID with parsing the list to be a real list not just stringified
+@router.get("/{metricID}", response_model=schemas.List[schemas.Metric])
+def getMetric(metricID: int, metricCrud: crud.CRUDMetric = Depends(dependencies.getMetricsCrud)) -> Any:
+    metric = metricCrud.get(metricID)
+    metric.area = json.loads(metric.area)
```

Comment:
json.loads may throw exception if database contains invalid data for metric.area for any reason.
make sure to handle exceptions and in case of any exception for the metric.are, return the metric data but with empty area field.